### DISCUSSION TITLE
Getting error with openai

### DISCUSSION THREAD

**Discussion by aravinth-numberz** [KruxAI/ragbuilder#18] [General]
I'm getting an error while running while creating new project

> [INFO] 2024-08-22 09:55:50 - common.py -   File "/usr/local/lib/python3.12/site-packages/anyio/_backends/_asyncio.py", line 2177, in run_sync_in_worker_thread
> [INFO] 2024-08-22 09:55:50 - common.py -     return await future
> [INFO] 2024-08-22 09:55:50 - common.py -            ^^^^^^^^^^^^
> [INFO] 2024-08-22 09:55:50 - common.py -   File "/usr/local/lib/python3.12/site-packages/anyio/_backends/_asyncio.py", line 859, in run
> [INFO] 2024-08-22 09:55:50 - common.py -     result = context.run(func, *args)
> [INFO] 2024-08-22 09:55:50 - common.py -              ^^^^^^^^^^^^^^^^^^^^^^^^
> [INFO] 2024-08-22 09:55:50 - common.py -   File "/usr/local/lib/python3.12/site-packages/ragbuilder/ragbuilder.py", line 317, in rbuilder_route
> [INFO] 2024-08-22 09:55:50 - common.py -     result = parse_config(project_data.model_dump(), db)
> [INFO] 2024-08-22 09:55:50 - common.py -              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> [INFO] 2024-08-22 09:55:50 - common.py -   File "/usr/local/lib/python3.12/site-packages/ragbuilder/ragbuilder.py", line 423, in parse_config
> [INFO] 2024-08-22 09:55:50 - common.py -     f_name=generate_data.generate_data(
> [INFO] 2024-08-22 09:55:50 - common.py -            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> [INFO] 2024-08-22 09:55:50 - common.py -   File "/usr/local/lib/python3.12/site-packages/ragbuilder/generate_data.py", line 106, in generate_data
> [INFO] 2024-08-22 09:55:50 - common.py -     f_name=f'rag_test_data_{critic_llm.model}_{ts}.csv'
> [INFO] 2024-08-22 09:55:50 - common.py -                             ^^^^^^^^^^^^^^^^
> [INFO] 2024-08-22 09:55:50 - common.py - AttributeError: 'ChatOpenAI' object has no attribute 'model'

**Comment by aravind10x** [comment]
Hi @aravinth-numberz sorry for the late reply! But this was fixed last month with this [PR](https://github.com/KruxAI/ragbuilder/pull/19). 


Can you please confirm that this issue is fixed?
